app/models/account_objective.rb

```diff
@@ -0,0 +1,7 @@
+class AccountObjective < ApplicationRecord
+  belongs_to :account
+  belongs_to :owner, class_name: 'User', foreign_key: 'owner_id'
+
+  validates :target_name, presence: true
```

Comment:
I think we don't need `target_name` because the account's name already has the objective.

For example, possible names for the account objectives persisted at `accounts.name`:

- New Car 2024
- New Playstation

![image](https://user-images.githubusercontent.com/14297526/186906919-3c3cd5d3-772a-4629-ac69-ae8e2b25de88.png)